src/ansi-c/ansi_c_entry_point.cpp

```diff
@@ -201,7 +218,7 @@ bool generate_ansi_c_start_function(
   init_code.add(code_labelt(CPROVER_PREFIX "HIDE", code_skipt()));
 
   // build call to initialization function
-
+  if(enable_static_lifetime_initialisation)
```

Comment:
I'm aware this is a draft, but just an early warning: there could be all sorts of places that (implicitly) expect that this call be present. The exact instructions in the function won't matter as much.

Reply:
Thanks. I have every expectation that there will be things which are broken and broken in surprising ways. What we are trying to avoid is a partial implementation which ignores (with a warning or not) kinds of expression which are not implemented. The idea is that we will halt at the first expression which is not understood, in order to make these easy to find rather than like looking for a needle in a haystack. The problem is that we are expecting unimplemented functionality in the initialise function. Therefore by skipping it we can still test rudimentary functionality in trivial user code examples, in regression tests. So once we reach the point where everything in the initialise function is supported we can revert skipping it and proceed with more real-world examples. We are using the fairly coarse approach of skipping the initialise function entirely, rather than filtering out unsupported features because this is a temporary solution only.

Reply:
This code is now gone. So that implicit expectation should no longer be an issue.